Example/Pods/Local Podspecs/PINRemoteImage.podspec.json

```diff
@@ -9,26 +9,48 @@
   },
   "source": {
     "git": "https://github.com/pinterest/PINRemoteImage.git",
-    "tag": 1.0
+    "tag": "1.1.2"
```

Comment:
I figured out why this was happening, after I rebased I noticed this was still happening, and it was because the tag was not updated in the PINRemoteImage.podspec file, so I updated it in my pull request.